app/Models/Follow.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+use Illuminate\Database\Eloquent\Model;
+
+class Follow extends Model
```

Comment:
```suggestion
class Follow extends Pivot
```
đây ko phải một model
với lại thay vì tạo quan hệ 1-n trong đây, bn nên tạo quan hệ m-n giữa 2 bảng chính thì hợp lý hơn.